src/styles/components/_header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  display: flex;
```

Comment:
And put it into `reset.scss` or `normalise.scss`
```
* {
  box-sizing: border-box
}
```

Reply:
Done